commcare/src/main/java/org/motechproject/commcare/service/impl/CommcareConfigServiceImpl.java

```diff
@@ -215,9 +215,8 @@ private void updateConfigs() {
     }
 
     private Map<String, Object> prepareParams(String name) {
-        Map<String, Object> params = new HashMap<>();
-        params.put(EventDataKeys.CONFIG_NAME, name);
-        return params;
+        boolean isConfigVerified = verifyConfig(configs.getByName(name));
+        return prepareParams(name, isConfigVerified);
```

Comment:
I see that having 2 prepareParams methods is unnecessary since you add verifyConfig. Use verifyConfig in syncConfig method